### DISCUSSION TITLE
Typescript error on Build (error TS7010: 'each', which lacks return-type annotation, implicitly has an 'any' return type.)

### DISCUSSION THREAD

**Discussion by jvUP** [GrapesJS/grapesjs#4917] [Q&A] [answered]
Hi, after version 0.20.1 I was getting a _typescript error_ on build. My last trying was with the 0.20.4 of _grapesjs_

to reproduce, I run this command: `yarn run tsc`

the output is:
```
node_modules/grapesjs/index.d.ts:29:5 - error TS7010: 'each', which lacks return-type annotation, implicitly has an 'any' return type.

29     each(callback: (item: TModel) => void);
       ~~~~
```

When I follow the line of error, I see this:
![image](https://user-images.githubusercontent.com/111390170/217315990-60f13c28-276e-42a7-a612-f6d42aafcea0.png)

My `tsconfig.json` is settings as below
```
{
  "compilerOptions": {
    "target": "ESNext",
    "useDefineForClassFields": true,
    "lib": ["DOM", "DOM.Iterable", "ESNext"],
    "allowJs": false,
    "skipLibCheck": false,
    "esModuleInterop": false,
    "allowSyntheticDefaultImports": true,
    "strict": true,
    "forceConsistentCasingInFileNames": true,
    "noImplicitReturns": true,
    "module": "ESNext",
    "moduleResolution": "Node",
    "resolveJsonModule": true,
    "isolatedModules": true,
    "noEmit": true,
    "jsx": "react-jsx",
    "baseUrl": ".",
    "paths": {
      "@/*": ["./src/*"]
    }
  },
  "include": ["src"],
  "references": [{ "path": "./tsconfig.node.json" }]
}
```

If I understand correctly, we need to explicit the return value as _**any**_, because now it is only an implicit _**any**_

I will appreciate If someone can fix it, or knows how to fix it, or help me to know how to fix this in a PR

**Comment by ptesser** [comment]
A possible solution / workaround is setting `skipLibCheck: true` in the `compilerOptions` property.

**Comment by artf** [comment, marked as answer]
This will by fixed in the next release https://github.com/GrapesJS/grapesjs/pull/4903
